spec/unit/rack/user_spec.rb

```diff
@@ -189,5 +189,19 @@ def current_user
         expect(user_data).to be_empty
       end
     end
+
+    context "when Rack user field getters require parameters" do
+      class UserRequiringParameters
```

Comment:
Would probably rephrase to something like "... user has methods with arguments". Then, would use a `let` block and an anonymous class as in other examples.